docs/int/quickstart/quickstart-group.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 5
+description: Run one node in a multi-operator distributed validator cluster
+---
+
+# Run a cluster with others
+
+:::caution
+Charon is in an early alpha state and is not ready to be run on mainnet
+:::
+
+To create a distributed validator cluster with a group of other node operators requires four key steps:
+- Every operator prepares their software and gets their charon client's ENR
+- One operator prepares the terms of the distributed validator key generation ceremony
+    - They select the network, the withdrawal address, the number of 32 ether distributed validators to create, and the ENRs of each operator taking part in the ceremony.
+    - In future, the DV launchpad will facilitate this process more seamlessly, with consent on the terms provided by all operators that participate.
+- Every operator runs the ceremony, and once successful, a number of cluster artifacts are created, including:
+    - The private key shares for each distributed validator
+    - The deposit data file containing deposit details for each distributed validator
+    - A `cluster-lock.json` file which contains the finalised terms of this cluster required by charon to operate. 
+- Every operator starts their node with `charon run`, and uses their monitoring to determine the cluster health and connectivity
+- Once the cluster is confirmed to be healthy, deposit data files created during this process are activated on the staking launchpad. 
+
+### Getting setup with charon
```

Comment:
```suggestion
# Getting started with a single distributed validator charon node
```
Something sort of like this to have a separate heading for running a single DV node